client/cli/src/client_proxy.rs

```diff
@@ -1119,7 +1119,7 @@ impl ClientProxy {
             format!("http://{}", self.faucet_server).as_str(),
             &[
                 ("amount", num_coins.to_string().as_str()),
-                ("address", format!("{:?}", receiver).as_str()),
+                ("address", &hex::encode(receiver)),
```

Comment:
Should this param name be changed to `autentication_key` (or TODO for changing it later)?